lib/pos_block_producer.go

```diff
@@ -347,7 +347,7 @@ func (pbp *PosBlockProducer) getBlockTransactions(
 				return nil, 0,
 					errors.New("Error casting txn meta to AtomicSwapMetadata")
 			}
-			blockProducerPublicKeyBtcec, err := btcec.ParsePubKey(blockProducerPublicKey.ToBytes())
+			blockProducerPublicKeyBtcec, err := btcec.ParsePubKey(blockProducerPublicKey.ToBytes(), btcec.S256())
```

Comment:
This diff isn't in #1163, but is correct. Assuming this bit of code was added AFTER deps upgrade was merged when adding support for atomic txns in the pos mempool.

Reply:
Correct. This code was added in https://github.com/deso-protocol/core/commit/729ff8376da15f302236df93cebce02967840bc0#diff-5e197e8b9753d456923289fea3939b21ab48d9f97c263ffda4f41a7f1f432be6. It needs to be updated to use the btcec v1 library